### DISCUSSION TITLE
LAMMPS failed with model trained with float64

### DISCUSSION THREAD

**Discussion by saeedshahi** [mir-group/pair_nequip#24] [Q&A] [answered]
Hi,

I trained an ML potential with float64 and it converged perfectly but when I want to use it with LAMMPS, it failed and I got this error "RuntimeError: expected scalar type Float but found Double".
the model trained with float32 works fine with LAMMPS.

Best,
Saeed

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @saeedshahi ,

Thank you for your interest in our code!

`pair_nequip` does not yet support float64 models (https://github.com/mir-group/pair_nequip/discussions/21). The link contains details about how a model trained in float64 can be deployed for float32 inference.
